include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -22,8 +22,8 @@ namespace seqan3
 
 /*!\brief The alphabet of a gap character '-'
  * \ingroup gap
- * \implements seqan3::Alphabet
- * \implements seqan3::detail::ConstexprAlphabet
+ * \implements seqan3::WritableAlphabet
```

Comment:
this is already contained in `WritableConstexprAlphabet`
Why do you name it explicitly, but not ConstexprAlphabet or Alphabet or Semialphabet?

Reply:
Because the ConstExpr ones are in detail and not visible. It's not very consistent, though, I agree.

Reply:
Is there a reason for hiding the constexpr alphabets from the user? We could make them visible and the concepts can be used.

Reply:
Well, there are so many concepts already for something that should be rather simple. I would prefer not to add four more base concepts to the user documentation.